File: presentation/06_FileIO/file_io.swift
// Topic: File I/O
// Write to test.txt

import Foundation

let str = "Hello Swift File IO"
let filename = "test.txt"

do {
    try str.write(toFile: filename, atomically: true, encoding: .utf8)
    print("Written to test.txt")
} catch {
    print("Failed to open file")
}

File: presentation/06_FileIO/file_io.py
# Topic: File I/O
# Write to test.txt

def main():
    try:
        with open("test.txt", "w") as f:
            f.write("Hello Python File IO")
        print("Written to test.txt")
    except IOError:
        print("Failed to open file")

if __name__ == "__main__":
    main()
